github-actions/docs/determine_if_we_need_to_build.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+$ref   = getenv('GITHUB_REF');
+$repo  = getenv('GITHUB_REPOSITORY');
+$token = getenv('GITHUB_TOKEN');
+
+function buildDocs() {
+    echo "TRUE";
+    exit(0);
+}
+
+function skipDocs() {
+    echo "FALSE";
+    exit(0);
+}
+
+/**
+ * @return string[]
+ */
+function executeApiCall(string $repo, string $resource, string $token): array
+{
+    curl_init('https://api.github.com/repos/' . $repo . '/' . $resource);
+    curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+    curl_setopt($curl, CURLOPT_HTTPHEADER, [
+        'Authorization: bearer ' . $token,
+        'User-Agent: laminas/docs',
+        'Accept: application/vnd.github+json',
+    ]);
+    $response = curl_exec($curl);
+    $payload  = json_decode($response, true);
+    $payload  = array_map(static function (string $ref): string {
+        return $ref['name'];
+    }, $payload);
+    usort($payload, $compareRefNames);
+    return $payload;
+}
+
+function extractMajorMinorVersion(string $version): string
+{
+    $matches = [];
+    preg_match('/^(?P<release>\d+\.\d+)\./', $version, $matches);
+    return $matches['release'] ?? $version;
+}
+
+if ('master' === $ref) {
+    // Assume that release branches are not in use if we detect a push to master
+    buildDocs();
+    exit(0); // redundant; placed here to note that buildDocs exits
+}
+
+if (! preg_match('/^\d+\.\d+\.x$/', $ref)) {
+    // Not a release branch
+    skipDocs(); // redundant; placed here to note that skipDocs exits
```

Comment:
I believe the additional comment should be on line with `exit(0)`  (as above), the same for couple more places below